Update readme clarifications (#13)
flightaware/aeroapps: README.md

You can test out the FIDS/Alerts sample application by visiting http://localhost:8080 in
your web browser (if not running Docker locally, use the Docker host's
address).

## Alerts Backend Note:

Please note that when you create an alert, whenever an event is triggered for that
alert, AeroAPI will send a POST request to your configured endpoint. In order to configure

> your endpoint, the server you are running the app on will need to have a publicly exposed

Reviewer: ```suggestion
your endpoint, you need to publicly expose its
```

address/port (using the POST_PORT environment variable, NOT the WEB_SERVER_PORT variable).

> Furthermore, as noted in the docker-compose-alerts.yml file, we encourage the service for

Reviewer: I think you can delete "Furthermore, as noted in the docker-compose-alerts.yml file, we encourage the service for accepting POSTed triggered alerts to be isolated, and thus will be sent to a different Docker
container. This means that you will have to set the endpoint URL using /post, instead of /api/post."

accepting POSTed triggered alerts to be isolated, and thus will be sent to a different Docker
container. This means that you will have to set the endpoint URL using /post, instead of /api/post.

> You can then use this address in setting the endpoint by sending a PUT request to AeroAPI

Reviewer: Then link documentation and give an example curl command

(see the [documentation](https://flightaware.com/aeroapi/portal/documentation#put-/alerts/endpoint)

> on how to set it). NOTE: if you already had a previously configured endpoint URL for other alerts,

Reviewer: ```suggestion
on how to set it). NOTE: if you previously configured a different production endpoint to receive alerts,
```

> **you will change the same URL!** This means that ALL of your configured alerts, including

Reviewer: ```suggestion
**you will change the same URL!** This means that ALL of your configured alerts
```

> ones configured on FlightAware's website, will all be sent to the newly configured endpoint.

Reviewer: ```suggestion
will all be sent to the newly configured endpoint.
```

You can see this newly configured URL by going to FlightAware's
[push notification testing page](https://flightaware.com/commercial/aeroapi/send.rvt)

> or by seeing it on the alert creation page on the webapp. On this push notification testing

Reviewer: ```suggestion
or by going to the alert creation page on the webapp. On this push notification testing
```

> page, you can also do two things: test your configured endpoint on the backend to see if it

Reviewer: Let's start a new paragraph detailing the use of the notification testing page, and break this up into a few sentences

(http://localhost:8081/post for example) to receive alerts using a REST client like cURL,
like the following command:
```
curl --location --request PUT 'https://aeroapi.flightaware.com/aeroapi/alerts/endpoint' \
--header 'Content-Type: application/json; charset=UTF-8' \
--header 'x-apikey: <YOUR API KEY>' \
--data-raw '{
  "url": "http://localhost:8081/post"
}'
```
(see the [documentation](https://flightaware.com/aeroapi/portal/documentation#put-/alerts/endpoint)
for more information). NOTE: if you previously configured a different production endpoint to receive alerts,
**you will change the same URL!** This means that ALL of your configured alerts
will all be sent to the newly configured endpoint.

You can see this newly configured URL by going to FlightAware's
[push notification testing page](https://flightaware.com/commercial/aeroapi/send.rvt)
or by going to the alert creation page on the webapp. On this push notification testing
page, you can also do two things: test your configured endpoint on the backend to see if it
receives a test triggered alert properly, and also see the success/failure of triggered alerts

> that were sent up to 24 hours in the past, and if there was an error the backend will send the

Reviewer: ```suggestion
that were sent up to 24 hours in the past. If there was an error it will display the
```

> proper message of that error. 

Reviewer: ```suggestion
full error message.
```

--header 'Content-Type: application/json; charset=UTF-8' \
--header 'x-apikey: <YOUR API KEY>' \
--data-raw '{

>   "url": "<YOUR IP>"

Reviewer: ```suggestion
  "url": "http://<YOUR IP>:8081/post"
```
